components/ReviewFeed.tsx

```diff
@@ -0,0 +1,20 @@
+import ReviewContainer from "./ReviewContainer";
+
+interface Review {
+  id: number;
+  text: string;
+  rating: number;
+}
+
+interface ReviewFeedProps {
+  reviews: Review[];
+}
+export const ReviewFeed: React.FC<ReviewFeedProps> = ({ reviews }) => {
+  return (
+    <div>
```

Comment:
do you need this div? can we have Fragment here as well? If yes, use Fragment :)